benchmarks/benchmarks/stats.py

```diff
@@ -381,29 +381,37 @@ def time_binned_statistic_dd_reuse_bin(self, statistic):
 
 class ContinuousFitAnalyticalMLEOverride(Benchmark):
     # list of distributions to time
-    dists = ["pareto", "laplace", "rayleigh",
-             "invgauss", "gumbel_r", "gumbel_l"]
+    dists = ["pareto", "laplace", "rayleigh", "invgauss", "gumbel_r",
+             "gumbel_l", "powerlaw"]
     # add custom values for rvs and fit, if desired, for any distribution:
     # key should match name in dists and value should be list of loc, scale,
     # and shapes
     custom_input = {}
     fnames = ['floc', 'fscale', 'f0', 'f1', 'f2']
     fixed = {}
-    distcont = dict(distcont)
+    distcont = distcont
 
-    param_names = ["distribution", "loc_fixed", "scale_fixed",
+    param_names = ["distribution", "shape_in_use", "loc_fixed", "scale_fixed",
                    "shape1_fixed", "shape2_fixed", "shape3_fixed"]
-    params = [dists, * [[True, False]] * 5]
-
-    def setup(self, dist_name, loc_fixed, scale_fixed, shape1_fixed,
-              shape2_fixed, shape3_fixed):
+    # `n_shape_iter` needs to be an enumeration of the maximum number of
```

Comment:
I don't see `n_shape_iter` anywhere.
If it is the same as `n_shape_use`, then I don't understand how `range(2)` is an "enumeration of the maximum number of shapes present in `_distr_params.py`". Does this mean "sequence of integers up to (but not including / and including?) the maximum number of shapes present..."? Should it be `range(3)` if some distributions have three shapes?

Reply:
I think I confused myself when making that. I intended that this would be updated when more distributions are added if they have more shapes. Right now all of the distributions that are being benchmarked only have at maximum 1 shape, so I thought that it would be efficient to not have benchmarks that are always skipped with 2 or 3 shapes. 

So right now it only needs to be `range(1)`, but I suppose it would be simpler just to make it `range(3)` in the first place to not have to worry about it later.